Cartfile

```diff
@@ -1,3 +1,4 @@
 github "attaswift/BigInt" ~> 3.1
 github "keefertaylor/TezosCrypto" ~> 2.1
 github "mxcl/PromiseKit" ~> 6.8
+github "pokt-network/pocket-swift-core-carthage"
```

Comment:
You likely also want to update TezosKit.podspec so that this library can be consumed via Cocoapods as well. Should be relatively easy, and we can verify everything works with `pod lib lint`